frontend/src/stores/hydrologic.js

```diff
@@ -7,9 +7,8 @@ export const useHydrologicStore = defineStore('hydrologic', () => {
   const swotVariables = ref([
     {
       abbreviation: 'time_str',
-      name: 'Time',
-      unit: '',
-      definition: 'Time of the measurement',
+      name: 'Date',
+      definition: 'Date and time of the measurement',
```

Comment:
I hesitate to change these `swotVariables` because up to now, they have conformed with the actual NASA schema/nomenclature:
https://archive.podaac.earthdata.nasa.gov/podaac-ops-cumulus-docs/web-misc/swot_mission_docs/pdd/D-56413_SWOT_Product_Description_L2_HR_RiverSP_20231026_RevB_w-sigs.pdf

Reply:
Perhaps we could have 2 properties, for example:
1. `name` (the NASA defined name)
2. `swotVisInternalName` (the value that we want to display in our app)

We could do the same for `definition` and other properties.


Or if that is too much trouble, we can do as you have here but we should realize that we are making a conscious decision to depart from the variables/terminology used by NASA/SWOT

Reply:
I guess maybe this is more trouble than it's worth. We gotta keep moving! :)
I'm fine with departing from the NASA terms as long as you are